test/client/setup-test-framework.js

```diff
@@ -44,7 +47,7 @@ jest.mock( 'wpcom-proxy-request', () => ( {
 } ) );
 
 // Mock crypto.randomUUID with its Node.js implementation
-global.crypto.randomUUID = () => require( 'crypto' ).randomUUID();
+global.crypto.randomUUID = () => nodeCrypto.randomUUID();
```

Comment:
Not opposed to using the node built-in one 👍

Reply:
Yep, substituted with that because we use the same `require` to import `subtle`.